server/src/main/scala/org/dbpedia/extraction/server/resources/rml/model/Prefixes.scala

```diff
@@ -0,0 +1,29 @@
+package org.dbpedia.extraction.server.resources.rml.model
+
+/**
+  * Contains the prefixes map
+  */
+object Prefixes {
```

Comment:
we should reuse the existing ns code from here
https://github.com/dbpedia/extraction-framework/blob/398bd582b3d2233531c8f3cb94270208392b0910/core/src/main/scala/org/dbpedia/extraction/ontology/RdfNamespace.scala#L30-L30